### ISSUE TITLE
Error in loadNamespace

### ISSUE THREAD

**Issue by kcduncan** [ablack3/icdpicr#2]
I tried to run ICDPICR on my data but got this error message:
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘stringi’

**Comment by ablack3** [comment]
I'm glad you are using the package! The package depends on a function in the stringr package which in turn depends on the stringi package. They should have been installed automatically when you installed icdpicr but apparently that didn't work. I'll look into why that didn't work. In the meantime I think the problem will be fixed if you just makes sure the stringr and stringi packages are installed.

First just try running
```
install.packages("stringr")
```
It you still get the error then run:
```
install.packages("stringi")
```
Let me know if that works.

**Comment by kcduncan** [comment]
Thanks so much Adam, I managed to get the online web-based version to work
for me. I had a quick question though, does the order of the diagnosis
codes matter for the ICDPICR algorithm? In my dataset, the principle
diagnosis is coded as prindiag and all the other diagnoses are coded diag1
diag2 etc. So I just generated a new diag (diag30) that is equal to my
principle diagnosis. However, does it matter that this is now diag30 and
not the first diagnosis?

Many thanks!!
Katrina
